spec/features/applications/show_spec.rb

```diff
@@ -58,5 +57,48 @@
       expect(page).to have_link(@pet_2.name)
       expect(page).to have_link(@pet_3.name)
     end
+
+    it "I can approve a pet's application" do
```

Comment:
made this into a long test - user story 23 (as far as I can tell) was satisfied by default, so I just added onto this test for us22 to cover us23 as well